desktop/src/app.rs

```diff
@@ -112,9 +113,7 @@ impl App {
                 }
 
                 // Core loop
-                winit::event::Event::MainEventsCleared
-                    if matches!(loaded, LoadingState::Loaded) =>
-                {
+                winit::event::Event::AboutToWait if matches!(loaded, LoadingState::Loaded) => {
```

Comment:
Do we want to mention in a comment that `AboutToWait` is not _actually perfectly_ what we'd want - as discussed on Discord -, and that we just cross our fingers, and most of the time it does work out? Not that important, it's just in our collective thought contexts now, but may not be later, _if_ it ends up causing weird issues.

Reply:
Yeah good call, I'll put a comment here. Given nosamu saw behaviour changes, my immediate thought is that this is likely the culprit already and we'll wanna revisit this soon

Reply:
Added comment